src/main/java/org/jahia/modules/portalFactory/esigate/flow/BeanUtils.java

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
you don't need to copy the class, either:
- import the library you want to use
- use one provided that do the same such as `org.apache.commons.beanutils.PropertyUtils`

Reply:
I've used the PropertyUtils as you suggest it